sql/core/src/test/resources/sql-tests/results/interval.sql.out

```diff
@@ -2292,49 +2292,55 @@ cannot resolve '(INTERVAL '1' MONTH > INTERVAL '20' DAY)' due to data type misma
 -- !query
 SELECT INTERVAL '1' DAY < '1'
```

Comment:
I wonder are there checks where the string at the left side?

Reply:
Added new test cases.